grpc/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[package]
+name = "celestia-grpc"
+version = "0.1.0"
+edition = "2021"
+license = "Apache-2.0"
+description = "A collection of traits for interacting with Celestia validator nodes gRPC"
+authors = ["Eiger <hello@eiger.co>"]
+homepage = "https://www.eiger.co"
+repository = "https://github.com/eigerco/lumina"
+readme = "README.md"
+# crates.io is limited to 5 keywords and 5 categories
+keywords = ["blockchain", "celestia", "lumina"]
+# Must be one of <https://crates.io/category_slugs>
+categories = [
+  "api-bindings",
+  "asynchronous",
+  "encoding",
+  "cryptography::cryptocurrencies",
+]
+
+[dependencies]
+celestia-tendermint-proto.workspace = true
+celestia-types = { workspace = true, features = [ "tonic" ] }
+celestia-proto = { workspace = true, features = [ "tonic", "cosmrs" ] }
+celestia-tendermint.workspace = true
+prost.workspace = true
+celestia-grpc-macros = { version = "0.1.0", path = "grpc-macros" }
+cosmos-sdk-proto = "0.26.0"
+
+hex = "0.4.3"
+serde = "1.0.215"
+cosmrs = "0.21.0"
```

Comment:
is this still used?

Reply:
No, removed cosmrs entirely now and moved to our protobuf definitions and types in celestia_types and celestia_grpc depending on whether it's generic or grpc specific respectively.